src/Juno/JunoClient.ts

```diff
@@ -162,6 +171,61 @@ export class JunoClient {
     return this.getNotebooks(`${this.getNotebooksUrl()}/gallery/public`);
   }
 
+  // will be renamed once feature.enableCodeOfConduct flag is removed
+  public async fetchPublicNotebooks(): Promise<IJunoResponse<IPublicGalleryData>> {
+    const input = `${this.getNotebooksAccountUrl()}/gallery/public`;
+    const response = await window.fetch(input, {
+      method: "PATCH",

```

Comment:
Is `PATCH` intended here?

Reply:
yes, I followed the pattern where all authenticated requests are sent as PATCH requests. Juno also accepts a patch request on this URL